CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 0.1.12
+- Update Cadence testing to support synchronous child workflows.
```

Comment:
Can you please update this to reflect the "what" of the change? Something like "Restore previous workflow context after finished processing". The main reason is — CHANGELOG is used by others to estimate the risk and the need of an upgrade. The current copy suggests that it's a change that only affects testing, but that's not true and the change is broader

The "why" is also very important and would go into the PR description.